### PR TITLE
Increased length of dcim.Device.serial to 255

### PR DESCRIPTION
**Pull request by briddo**
### Database Changes: migration file created

### CODE REVIEW

**Review by jathanism — APPROVED** [review]
Works for me! Thanks! :shipit:

**Review by glennmatthews — COMMENTED** [review]
As noted in #909, it would probably be appropriate to increase the serial number length on InventoryItem and Rack models as well.

Also, looks like the corresponding form fields in nautobot/dcim/forms.py also need to be updated to allow for the increased length as an input.

**Comment by glennmatthews** [comment]
It would be good to adjust the test data in nautobot/dcim/tests/test_views.py to exercise the increased limits as well.

**Comment by briddo** [comment]
> It would be good to adjust the test data in nautobot/dcim/tests/test_views.py to exercise the increased limits as well.

OK I'll do this a little later

**Comment by briddo** [comment]
> It would be good to adjust the test data in nautobot/dcim/tests/test_views.py to exercise the increased limits as well.

This is now done

File: nautobot/dcim/migrations/0009_increase_length_on_all_serials.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.14 on 2021-12-21 20:29
```
Comment (glennmatthews) [inline nautobot/dcim/migrations/0009_increase_length_on_all_serials.py:1]: No need for these to be two separate migrations files - can you please delete and regenerate a single combined migration?

**Review by glennmatthews — APPROVED** [review]
Almost there, thanks!

**Pull request merged**